Remotion/Web/Core/Web.Core.csproj

```diff
@@ -31,5 +31,8 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'net48'">
     <Reference Include="System.Configuration" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="..\Development.Analyzers\bin\Debug\netstandard2.0\Remotion.Web.Development.Analyzers.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```

Comment:
We should use a variable for the configuration so that we support `Debug` and `Release`. The variable name is `$(Configuration)`